### PR TITLE
Adding note for new node creation issue (BSC#1174449, BSC#1174449)

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/admin_saltcluster.xml [hunk 1]
```diff
@@ -69,6 +69,14 @@
      </para>
     </step>
    </procedure>
+   <note>
+     <para>
+       If this is <emphasis>not</emphasis> a new node, but you want to proceed as if
+       it was, ensure you remove the <filename>/etc/ceph/destroyedOSDs.yml</filename> file
```
Comment (lproven) [inline xml/admin_saltcluster.xml:75]: To be formally correct: "as if it were".

Comment (asettle) [inline xml/admin_saltcluster.xml:75]:
```suggestion
       If this is <emphasis>not</emphasis> a new node, but you want to proceed as if
       it were, ensure you remove the <filename>/etc/ceph/destroyedOSDs.yml</filename> file

File: xml/admin_saltcluster.xml [hunk 2]
```diff
@@ -61,11 +67,30 @@
...
+</screen>
+      </note>
     </step>
     <step>
      <para>
       After the new OSDs are added, adjust their weights as required with the
-      <command>ceph osd reweight</command> command.
+      <command>ceph osd reweight</command> command in small incraments. This
+      allows the cluster to rebalance and become healthy between increasing
+      incraments so it does not overwhelm the cluster and clients accessing
```
Comment (swiftgist) [inline xml/admin_saltcluster.xml:92]: same. :)

File: xml/admin_saltcluster.xml [hunk 3]
```diff
@@ -61,11 +67,30 @@
...
+&prompt.smaster;salt-run state.orch ceph.stage.2
+&prompt.smaster;salt 'node*' state.apply ceph.osd
+</screen>
+      </note>
     </step>
     <step>
      <para>
       After the new OSDs are added, adjust their weights as required with the
-      <command>ceph osd reweight</command> command.
+      <command>ceph osd reweight</command> command in small incraments. This
```
[the remark this answers is not in the document]
Comment (asettle) [reply xml/admin_saltcluster.xml:90]: See, this is why you don't just copy-pasta

**Review by swiftgist — APPROVED** [review]
Besides the two misspellings, looks good.

**Review by tbazant — APPROVED** [review]
lgtm

**Comment by asettle** [comment]
Need to "forward port" into SES 6

**Pull request merged**